packages/core/admin/admin/src/content-manager/history/pages/History.tsx

```diff
@@ -109,22 +136,24 @@ const HistoryPage = () => {
             defaultMessage: '{contentType} history',
           },
           {
-            contentType: layout.contentType.info.displayName,
+            contentType: displayName,
           }
         )}
       />
       <HistoryProvider
         contentType={slug}
         id={documentId}
+        schema={schema}
         layout={layout}
         selectedVersion={selectedVersion}
         versions={versionsResponse.data}
         page={page}
+        mainField={mainField}
       >
         <Flex direction="row" alignItems="flex-start">
           <Main grow={1} height="100vh" overflow="auto" labelledBy={headerId}>
             <VersionHeader headerId={headerId} />
-            <VersionContent />
+            <VersionContent permissions={permissions} />
```

Comment:
do you need to pass permissions? why not lift the DocumentRBAC provider out?

Reply:
why not indeed!